test/run-integration-tests.sh

```diff
@@ -116,4 +116,17 @@ else
     echo -e "\nTOTAL FAILURES: $FAILURES OF $COUNT"
 fi
 
+# If we have coverage files, merge them into a single report for upload
+if [ -n "${ARTIFACT_DIRECTORY}" ]; then
+    lcov_args=""
```

Comment:
```suggestion
    lcov_args=()
```

(With `lcov_args=""; lcov_args+=(...)` it adds a first empty element to the array.)

Otherwise LGTM

Reply:
fixed - setting green label given the approval, thanks for having a look